Dolphin scripts/README.md

```diff
@@ -4,7 +4,18 @@
 2. Turn on "Scripting" logs and set Verbosity to "Error" or any option below (not "Notice").
 3. Under "Scripts", click "Add new Scripts" and select your Python script.
 
-## Entrance Randomizer (prototype)
+## Entrance Randomizer
+
+### Features
+- Randomization of almost all basic transitions in the game
+- Randomization of Shaman Shop prices
+- Generation of Spoiler Log .txt file
+- Generation of Randomized Map .graphml file
+- Option to manually choose a seed, or to get a random seed
+- Option to manually choose a starting area, or to get a random starting area
+- Option to make all transitions 2-directional, or to allow 1-directional transitions
+- Option to disable maps from the Shaman Shop, or to let them remain
```

Comment:
```suggestion
- Option to disable maps from the Shaman Shop
```

Reply:
Why change this?
The 3 options above describe 2 variants each, why not do this for all 4?

Reply:
Idk, it felt off at the time. Like too self-obvious. It's fine.